structure/structure_test.go

```diff
@@ -53,7 +53,11 @@ func (s *testTxStructureSuite) TearDownSuite(c *C) {
 func (s *testTxStructureSuite) TestString(c *C) {
 	txn, err := s.store.Begin()
 	c.Assert(err, IsNil)
-	defer txn.Rollback()
+	defer func() {
+		// to fix: invalid txn
+		err = txn.Rollback()
```

Comment:
We commit this `txn` at the end of this function. So I think we can remove this code.